wandb/sdk/internal/file_stream.py

```diff
@@ -292,6 +291,8 @@ def _thread_except_body(self):
             self._exc_info = exc_info
             logger.exception("generic exception in filestream thread")
             util.sentry_exc(exc_info, delay=True)
+            message = "File stream thread has met a critical exception: {}".format(e)
+            self._api.retry_callback(0, message)
```

Comment:
nit, If this file was properly typed it would tell you to do:
```
if self._api.retry_callback:
            message = "File stream thread has met a critical exception: {}".format(e)
            self._api.retry_callback(0, message)
```